### ISSUE TITLE
Crash Failed to find accessibility element with the label

### ISSUE THREAD

**Issue by PatrickRguez** [kif-framework/KIF#851]
Hello.

Before some update, I could enter in a button without accessibilityLabel. If my button had title: "Enter" for example,  this worked for me. Now I have to enter the accessibilityLabel mandatorily, and now when I want to back in my navigationController, i get an error:

Failed to find accessibility element with the label "Back"

How can I solve this?

Thank you very much

**Comment by PatrickRguez** [comment]
KIF 3.4.1 works

**Comment by phatmann** [comment]
Are you using HEAD or version 3.4.2?

**Comment by PatrickRguez** [comment]
I always install KIF with cocoaPods:

target 'Acceptance Tests' do
  pod 'KIF', '~> 3.0', :configurations => ['Debug']
end

Now, I have to use:

target 'Acceptance Tests', :exclusive => true do
  pod 'KIF', '3.4.1', :configurations => ['Debug']
end

**Comment by phatmann** [comment]
So it has to be one of [these commits](https://github.com/kif-framework/KIF/compare/v3.4.1...master) that broke this.

**Comment by odedgold** [comment]
Hi, not sure if it's the same issue, but KIF fails to find label when running on real device.
More specifically - KIF does find the same label when running on simulator, running the same scenario. Any ideas why?
